hpc/app/workflows.yml

```diff
@@ -29,6 +29,7 @@ node_types:
           start:
             inputs:
               ansible_user: { type: string, default: { get_property: [SELF, host, username] } }
+              ssh_host:     { type: string, default: { get_property: [SELF, host, ssh_host] } }
```

Comment:
This input can be accessed via { get_attribute: [SELF, host, public_address] } as explained below